src/olympia/ratings/tests/test_views.py

```diff
@@ -185,22 +185,18 @@ def test_list_addon_queries(self):
 
         assert Rating.unfiltered.count() == 3
 
-        with self.assertNumQueries(5):
-            # 5 queries:
+        with self.assertNumQueries(7):
+            # 7 queries:
```

Comment:
**not** approved :hammer: :see_no_evil:

Reply:
Actual count without the mocks I removed before this patch is like 13 or something, since we were calling all addon transformers for not good reason :scream:

Reply:
Hehe